themes/10up-theme/tailwind.config.js

```diff
@@ -10,32 +34,60 @@ module.exports = {
 	 */
 	content: [
 		/* Ensure changes to all PHP, JS, and JSON files rebuild your CSS */
-		'./assets/js/**/*.js',
-		'./includes/**/*.{php,js,json}',
-		'./partials/**/*.php',
-		'./templates/**/*.php',
+		'404.php',
+		'archive.php',
+		'attachment.php',
+		'author.php',
+		'category.php',
+		'date.php',
+		'footer.php',
+		'front-page.php',
+		'functions.php',
+		'header.php',
+		'home.php',
+		'index.php',
+		'page.php',
+		'search.php',
+		'searchform.php',
+		'single.php',
```

Comment:
Unfortunately, that is not possible due to the issues with webpack's glob handling that I outlined previously in that word doc you made for us. See here for additional details: https://v2.tailwindcss.com/docs/just-in-time-mode#styles-rebuild-in-an-infinite-loop

In short, if we do this in the root directory it causes an infinite loop (which is the original real I had the problem and @Nikki-Jones didn't, since she was targeting subfolders with her globs). 

Until we can find a way to address that or otherwise not use Webpack, I think that's the best we'll be able to do without a pretty big lift and definitely what we'd want to do for the MVP.

Reply:
We can't currently do this due to the way webpack handles globs, which I had noted before in the document you started originally. See here for details: https://tailwindcss.com/docs/content-configuration#styles-rebuild-in-an-infinite-loop

For our MVP, I'd say we need to consider this a documentation/training point and leave it at that. The only way around this is to fundamentally address the way the globs are handled or otherwise not use webpack. Neither of these things are small LOE or straightforward, and the effort for the dev to just do it and have it referenced in the README seems much smaller by comparison.

Reply:
Awesome, lets err on the side of simplicity for the Beta. Thanks @iansvo !